### ISSUE TITLE
Access getLastEventID method

### ISSUE THREAD

**Issue by codyjames** [born05/craft-sentry#11]
I'm trying to figure out how to implement this: https://docs.sentry.io/platforms/php/enriching-events/user-feedback/ but need to get access to the getLastEventID method. Is that accessible via this plugin?

Let me know.
Thanks!

**Comment by roelvanhintum** [comment]
Currently not, but you could access it via custom module's variable class. This has to be used from a template right? Sentry is accessible anywhere you `use Sentry;`. :)

**Comment by roelvanhintum** [comment]
@codyjames, sorry for the late response. `Sentry\State\Hub::getCurrent()->getLastEventId();` doesn't seem to work. Maybe outdated docs? I'd be happy to make this available from the plugin itself, but I cannot get `getLastEventId` to work.

**Comment by codyjames** [comment]
Right, I tried a similar approach but was not successful either. Not a big deal for me right now.

**Comment by roelvanhintum** [comment]
@codyjames does `Sentry\State\HubAdapter::getInstance()->getLastEventId()` work for you? I didn't get a result, but at least no errors.

**Comment by codyjames** [comment]
No, it did not work.
